eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -60,61 +60,60 @@ services:
+            - [setSiteAccess, ["@ezpublish.siteaccess"]]
         tags:
             - { name: kernel.event_subscriber }
 
     ezpublish.controller.base:
         class: %ezpublish.controller.base.class%
         abstract: true
         calls:
-            - [ setContainer, [@service_container] ]
+            - [ setContainer, ["@service_container"] ]
 
     ezpublish.controller.content.view:
         class: %ezpublish.controller.content.view.class%
         arguments:
-            - @ezpublish.view_manager
-            - @security.authorization_checker
+            - "@ezpublish.view_manager"
+            - "@security.authorization_checker"
         parent: ezpublish.controller.base
 
     ezpublish.controller.content.preview.core:
         class: %ezpublish.controller.content.preview.class%
         arguments:
-            - @ezpublish.api.service.content
-            - @http_kernel
-            - @ezpublish.content_preview_helper
-            - @security.authorization_checker
-            - @ezpublish.content_preview.location_provider
-            - @ezpublish.view.custom_location_controller_checker
+            - "@ezpublish.api.service.content"
+            - "@http_kernel"
+            - "@ezpublish.content_preview_helper"
+            - "@security.authorization_checker"
+            - "@ezpublish.content_preview.location_provider"
+            - "@ezpublish.view.custom_location_controller_checker"
 
     ezpublish.controller.content.preview:
         alias: ezpublish.controller.content.preview.core
 
     ezpublish.controller.content.download:
         class: %ezpublish.controller.content.download.class%
         arguments:
-            - @ezpublish.api.service.content
-            - @ezpublish.fieldType.ezbinaryfile.io_service
-            - @ezpublish.translation_helper
-            - @router
-            - @ezpublish.route_reference.generator
+            - "@ezpublish.api.service.content"
+            - "@ezpublish.fieldType.ezbinaryfile.io_service"
+            - "@ezpublish.translation_helper"
+            - "@router"
+            - "@ezpublish.route_reference.generator"
         parent: ezpublish.controller.base
 
     ezpublish.controller.content.download_redirection:
         class: %ezpublish.controller.content.download_redirection.class%
         arguments:
-            - @ezpublish.api.service.content
-            - @router
-            - @ezpublish.route_reference.generator
+            - "@ezpublish.api.service.content"
+            - "@router"
+            - "@ezpublish.route_reference.generator"
         parent: ezpublish.controller.base
 
     ezpublish.controller.page.view:
         class: %ezpublish.controller.page.view.class%
-        arguments: [@ezpublish.view_manager, @ezpublish.fieldType.ezpage.pageService]
+        arguments: ["@ezpublish.view_manager", "@ezpublish.fieldType.ezpage.pageService"]
         parent: ezpublish.controller.base
-        scope: request
```

Comment:
@pspanja, @andrerom: `scope` key is deprecated in Symfony 2.8, but I'm not sure how removing this here affects web application. I didn't notice any problems so far, but I want to make sure :)